CONTRIBUTING.md

```diff
@@ -33,7 +34,34 @@ The following text equips you with knowledge that makes contributing to ilastik
 
    ```
    conda config --set channel_priority strict
-   conda install --name base -c conda-forge mamba
+   conda install mamba -n base -c conda-forge
+   ```
+
+   If [strict channel priority][strict-channel-priority] does not allow installation of [Mamba][mamba]:
+
+   ```
+   conda config --set channel_priority false
+   conda install mamba -n base -c conda-forge
+   ```
+
```

Comment:
In general we would like to keep our Contributing guidelines a help for setting up ilastik development environments. While it might be helpful in some situations, I think troubleshooting conda/mamba should be done in the appropriate places. For this particular suggestions I would also want to add that not using strict channel priority is almost always a bad idea, as you might pull together incompatible builds.